cmd/geth/chaincmd.go

```diff
@@ -425,7 +425,8 @@ func exportPreimages(ctx *cli.Context) error {
 	if len(ctx.Args()) < 1 {
 		utils.Fatalf("This command requires an argument.")
 	}
-	stack, _ := makeFullNode(ctx)
+
```

Comment:
@karalabe is this type of extra space fine or not necessary because of the `}` ?

Reply:
I'll leave it up to you :) 

In general I only add an extra empty line if there's something super different coming up, but in that case I also add a comment :) All in all I don't want to micro manage people at this insanely low level, I just don't want comments to merge together with code :)